addon/components/feature-controls.js

```diff
@@ -20,34 +23,40 @@ export default Component.extend({
   _normalizeFlag(key) {
     return get(this, 'features._normalizeFlag')(key);
   },
+  // Refresh the state of the feature flags list component
   refresh() {
+    // Take the existing flags from the config and put them in a list of default values
     let featureFlags = this.get('featureFlags');
-    // Compute default values
     let defaults = {};
     for (let key in featureFlags) {
       defaults[this._normalizeFlag(key)] = featureFlags[key];
     }
-    
-    // Computed property is not possible, model is a local copy of feature flags
+    // Model is a local copy of the list of flags register for features service, used to compute properties on the full list
     let model = (get(this, 'features.flags') || []).map(key => {
       let meta =
-        ((featureControls && featureControls.metadata) || []).find(obj => {
+        ((featureControls && this.get('featureControls.metadata')) || []).find(obj => {
           return this._normalizeFlag(obj.key) === key;
         }) || {};
+      let isFlagLS = this.get('featureControls.useLocalStorage') && (this.get(`featuresLS.${key}`) !== undefined);
       let featureFlag = {
         key,
-        isEnabled: get(this, 'features').isEnabled(key),
+        isEnabled: isFlagLS ? this.get(`featuresLS.${key}`) : get(this, 'features').isEnabled(key),
         default: defaults[key] || false
       };
       return assign({}, meta, featureFlag);
     });
     set(this, 'model', model);
   },
   reset() {
+    // Reset the flags from the features service to the default value in the config
     let featureFlags = this.get('featureFlags');
     Object.keys(featureFlags).forEach(key => {
       this.updateFeature(this._normalizeFlag(key), featureFlags[key]);
     });
+    // If we use local storage then we want to clear the stored data
+    if(this.get('featureControls').useLocalStorage) {
```

Comment:
prefer `this.get('featureControls.useLocalStorage')` as above.

Reply:
I now remember that we tried with @GreatWizard to use this syntax but it wouldn't work. I have no explanation why...

Reply:
it's used a few lines above... 🤔